Justfile

```diff
@@ -56,6 +56,9 @@ build-server: lint-server
 test: test-web
 
 test-web: destroy-all launch-web && destroy-all
+  sleep 20  # Wait for server to launch.
```

Comment:
Maybe check that the port is accepting connections rather than sleep 20..

```bash
host=127.0.0.1
port=1337
interval=1

while ! nc -z "$host" "$port" > /dev/null 2>&1; do
    sleep "$interval"
done
```